### DISCUSSION TITLE
Non-unique usernames

### DISCUSSION THREAD

**Discussion by Ashe** [stoatchat/discussions#181] [Feature Suggestions]
I really like it when services don't care about your username. I like that discord has a code after your username since you know you can just have whatever username you want without issue. It's just another point of friction, if people don't get the username they want, they might spend time thinking about what they'd want to be instead and eventually forget to actually make an account altogether.

I don't know what the situation is with federation, multiple server instances, profile URLs etc.. but I wonder if we can make it happen somehow? Like, maybe if you signed up with your username on a different instance, you could be `MyUsername@MyInstance` where MyInstance is hidden unless you examine the user.

**Comment by Ashe** [comment]
I don't know how to delete this post since I've just found out that this feature is called **Discriminators** and is already being discussed in #61 - feel free to mark as duplicate / delete.

**Comment by DeclanChidlow** [comment]
https://revolt.chat/updates/evolving-usernames
